### PR TITLE
Adds the (lesser) predicted potion bundles for wizards.

### PR DESCRIPTION
**Pull request by Irradiation**
Same as the other potion bundles, except you get half as much potions (25 for the big bundle, 5 for the lesser bundle). On the other hand, you at least can tell what potions you have instead of murky bundles.

:cl:
 * rscadd: Added (lesser) predicted potion bundles for wizards.

### CODE REVIEW

File: code/game/gamemodes/wizard/artifacts.dm [hunk 1]
```diff
@@ -92,6 +92,19 @@
...
+/datum/spellbook_artifact/predicted_potion_bundle
+	name = "Predicted potion bundle"
+	desc = "Contains 25 potions. I like the blue ones myself."
+	abbreviation = "LPB"
+	price = 5 * Sp_BASE_PRICE
+	spawned_items = list(/obj/item/weapon/storage/bag/potion/predicted_potion_bundle)
+
+/datum/spellbook_artifact/lesser_predicted_potion_bundle
+	name = "Lesser predicted potion bundle"
+	desc = "Contains 5  potions. Don't go using them all in one place!."
```
Comment (DamianX) [inline code/game/gamemodes/wizard/artifacts.dm]: !.

File: code/game/objects/items/weapons/storage/bags.dm [hunk 2]
```diff
@@ -506,3 +506,23 @@ obj/item/weapon/storage/bag/plasticbag/quick_store(var/obj/item/I)
...
+
+/obj/item/weapon/storage/bag/potion/predicted_potion_bundle/New()
+	..()
+	for(var/i = 1 to 25)
+		var/potiontype = pick(existing_typesof(/obj/item/potion))
+		new potiontype(src)
+
+/obj/item/weapon/storage/bag/potion/lesser_predicted_potion_bundle
+	name = "Lesser predicted potion bundle"
+	desc = "What could potionly go slightly more right?."
```
Comment (DamianX) [inline code/game/objects/items/weapons/storage/bags.dm]: ?.

File: code/game/gamemodes/wizard/artifacts.dm (hunk 1 above)
Comment (DamianX) [reply code/game/gamemodes/wizard/artifacts.dm]: also a rogue space after `5`

File: code/game/objects/items/weapons/storage/bags.dm (hunk 2 above)
Comment (Irradiation) [reply code/game/objects/items/weapons/storage/bags.dm]: REEEE

**Comment by Shadowmech88** [comment]
For just 25 potions, are you sure that 100 spell points is a fair price?
If you take the number of all existing potions and their prices, what is the average total price of 25 randomly selected potions? Many of the potions are only 1 or 5 points each.

**Comment by Irradiation** [comment]
Haha, time for shit maths:

-There are 24 potions currently.
-The sum of their points amount to 136 points.
-The average of point value per potion is 5.67 points
-Ergo, the potion bundle of 25 potions has an average value of 141.75 points.

That might look big, but well, potions are one-use, unlike spells. SPEAKING OF:

-There are 37 spells currently.
-The sum of their points amount to 585 points.
-The average of point value per potion is 15.81 points
-Ergo, the spell bundle of 7 spells has an average value of  110.67 points.

I'm lazy and I decided to run 2 tests for 25-potion bundles and I got:
-Sum: 152 & 107
-Average value: 6.08 & 4.28

I do suppose I could jack up the potion count for the full bundle to 40 and the lesser bundle to 8 potions, unless that just isn't high enough. Potion buffs are to come maybe possibly.

**Comment by Caboozles** [comment]
I think if you're going pure random, the numbers going higher wouldn't hurt. Or even lowering the cost to 80 so they can pick up ONE spell. 

A potion seller gimmick with a mobility spell wouldn't be too bad, because lord knows how many people would valid you on sight.

**Comment by Irradiation** [comment]
You can buy the lesser bundle for the same price ratio as the 100 points bundle, so that's not really necessary.

**Comment by Caboozles** [comment]
Having eaten a sword, a potion of fullness, and a fireball right in a row, I will love there to be a potion bundle that does not result in that.

File: code/game/objects/items/weapons/storage/bags.dm [hunk 3]
```diff
@@ -506,3 +506,23 @@ obj/item/weapon/storage/bag/plasticbag/quick_store(var/obj/item/I)
 	..()
 	for(var/i=1 to 10)
 		new /obj/item/potion/random(src)
+
+/obj/item/weapon/storage/bag/potion/predicted_potion_bundle
+	name = "Predicted potion bundle"
+	desc = "What could potionly go right?."
```
Comment (MadmanMartian) [inline code/game/objects/items/weapons/storage/bags.dm:512]: Dat dot tho

**Pull request merged**